lib/core/viewmodels/dashboard_viewmodel.dart

```diff
@@ -0,0 +1,24 @@
+// FLUTTER / DART / THIRD-PARTIES
+import 'package:flutter/cupertino.dart';
```

Comment:
It's more a question for anyone who knows, but I've seen the cupertino and material import, it can be used to import the `@required` in ctor, is there any difference between those two imports?

Reply:
I'm not sure about the difference either, it seems https://flutter.dev/docs/development/ui/widgets/cupertino is the IOs style widget library but maybe @AlexandreMarkus knows more about that ? I'll be replacing this with the material.dart import as cupertino imports seem to have been removed from other widgets as well.